xep-0156.xml

```diff
@@ -154,12 +160,9 @@
   </revision>
 </header>
 <section1 topic='Introduction' anchor='intro'>
-  <p>Although &xmppcore; specifies the use of TCP as the method of connecting to an XMPP server, alternative connection methods exist, including the &xep0124; method (for which &xep0206; is the XMPP profile) and the websocket subprotocol specified in &rfc7395;. For some of these methods, it is necessary to discover further parameters before connecting, such as the HTTP URL of an alternative connection manager. Without ways to auto-discover alternative connection methods, the relevant information would need to be provided manually by a human user (which is cumbersome and error-prone) or hard-coded into XMPP software applications (which is brittle and not interoperable).</p>
-  <p>This document defines two ways to encapsulate information about alternative connection methods for auto-discovery:</p>
-  <ol>
-    <li>DNS TXT resource records</li>
-    <li>Link entries in a server's "host-meta" file</li>
-  </ol>
+    <p>The proper and secure way to look up websocket/bosh connections is defined in &rfc7395; Section 4, which is to grab the host-meta over HTTPS, clients/servers MUST only use this method.</p>
```

Comment:
RFC 7395 doesn't define bosh lookups

What about we don't obsolete this XEP but remove `_xmppconnect TXT records` ?

Reply:
Nevermind - happy to remove BOSH and XEP-0156 from XEP-0459: XMPP Compliance Suites 2022

https://mail.jabber.org/pipermail/standards/2022-February/038766.html

Reply:
I went ahead and just removed the DNS method, let me know how that looks